src/index.html

```diff
@@ -11,8 +11,64 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main>
+      <section
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__img"></div>
+
+        <div class="card__info">
+          <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+
+          <div class="card__code">Product code: 195434</div>
```

Comment:
```suggestion
          <p class="card__code">Product code: 195434</p>
```
Consider leveraging semantic tags for the text